src/test/java/com/epam/idea/rest/controller/TagControllerClass.java

```diff
@@ -0,0 +1,104 @@
+package com.epam.idea.rest.controller;
+
+import com.epam.idea.core.model.Tag;
+import com.epam.idea.core.model.builders.TestTagBuilder;
+import com.epam.idea.core.service.TagService;
+import com.epam.idea.core.service.exception.TagDoesNotExistException;
+import com.epam.idea.rest.config.TestConfig;
+import com.epam.idea.rest.config.WebAppConfig;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+
+import static com.epam.idea.rest.TestUtils.APPLICATION_JSON_UTF8;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+import java.util.Arrays;
+
+import static com.epam.idea.rest.resource.support.JsonPropertyName.ID;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.*;

```

Comment:
Do not use wildcard imports. See [code style](https://github.com/EPAMMogilev/idea/wiki/Java-Code-Style-Guideline#no-wildcard-imports).